python/metric_pipeline_utils/phot_repeat.py

```diff
@@ -111,6 +113,8 @@ def calcPhotRepeat(matches, magKey, numRandomShuffles=50):
     >>>     return np.isfinite(cat.get(magKey)).all()
     >>> repeat = calcPhotRepeat(allMatches.where(matchFilter), magKey)
     """
+    global getRandomDiff
+    getRandomDiff = functools.partial(getRandomDiff, rng=np.random.default_rng(seed))
```

Comment:
I'm a little concerned about using a global here to make this function show up in other function's scopes.  If we do go this route, I think it would be better to just pass the seed and set the seed in the function rather than relying on `np.random.default_rng(None)` returning None.  I'm not sure it does.

Reply:
Oh, and also seed is misspelled, it should be `randomSeed`